src/main/java/net/datafaker/providers/base/DateAndTime.java

```diff
@@ -264,6 +273,25 @@ public Timestamp birthday(int minAge, int maxAge) {
             LocalDateTime.of(date, LocalTime.ofNanoOfDay(faker.number().numberBetween(0, DAYS_NANOS))));
     }
 
+    /**
+     * Generates a random birthday between two ages from now as LocalDate.
+     *
+     * @param minAge the minimal age
+     * @param maxAge the maximal age
+     * @return a random birthday between {@code minAge} and {@code maxAge} years ago from now.
+     * @throws IllegalArgumentException if the {@code maxAge} is lower than {@code minAge}.
```

Comment:
Does this need to try/catch/rethrow?

Reply:
good catch, seem it is outdated and not related anymore to this method